test/models/test_inspection.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+"""
+<Program Name>
+  test_inspection.py
+
+<Author>
+  Santiago Torres-Arias <santiago@nyu.edu>
+
+<Started>
+  Nov 18, 2016
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Test inspection class functions.
+
+"""
+
+import unittest
+from toto.models.layout import Layout, Inspection
+from toto.ssl_commons.exceptions import FormatError
+import toto.ssl_crypto
+
+class TestInspectionValidator(unittest.TestCase):
+  """Test verifylib.verify_delete_rule(rule, artifact_queue) """
+
+  def setUp(self):
+    """ Populate a base layout that we can use """
+    self.inspection = Inspection("some-inspection")
+
+  def test_wrong_type(self):
+    """ test the type field within Validate() """
+
+    with self.assertRaises(FormatError):
+      self.inspection._type = "wrong"
+      self.inspection._validate_type()
+      self.assertFalse(self.inspection.validate())
+
+    self.inspection._type = "inspection"
+    self.inspection._validate_type()
+
+  def test_wrong_material_matchrules(self):
+    """ test that the material matchrule validators catch malformed ones """
+
+    with self.assertRaises(FormatError):
+      self.inspection.material_matchrules = [["NONFOO"]]
+      self.inspection._validate_material_matchrules()
+      self.assertFalse(self.inspection.validate())
+
+      self.inspection.material_matchrules = "PFF"
+      self.inspection._validate_material_matchrules()
+      self.assertFalse(self.inspection.validate())
```

Comment:
You only get until [line 48](https://github.com/in-toto/in-toto/pull/48/files#diff-ba23fca1006b681698d19e43ee88da7eR48). You need a separate `with self.assertRaises(FormatError)` for each exception that gets raised. This is a re-occurring thing in the unit tests below.